servers/rendering/shader_compiler.cpp

```diff
@@ -1107,7 +1107,9 @@ String ShaderCompiler::_dump_node_code(const SL::Node *p_node, int p_level, Gene
 					} else {
 						code += "";
 					}
-					code += _dump_node_code(cnode->array_declarations[0].initializer[i], p_level, r_gen_code, p_actions, p_default_actions, p_assigning);
+					if (i < cnode->array_declarations[0].initializer.size()) {
+						code += _dump_node_code(cnode->array_declarations[0].initializer[i], p_level, r_gen_code, p_actions, p_default_actions, p_assigning);
+					}
```

Comment:
I would remove this check - if it fails - it's signification that something bad on the shader_language side, and it needs to be fixed there.

Reply:
Done